client/stylesheets/_toCat.scss

```diff
@@ -2436,21 +2454,56 @@ i:hover {
     }
 }
 
+.headerDropDown ul li:first-child {
+	border-top: none !important;
+	margin-top: 10px;
+}
 
+.headerDropDown ul li:last-child {
+	border-bottom: none !important;
+	margin-bottom: 10px;
+} 
+
+.clickDropper li ul {
+	margin-left: -180px;
+}
+
+.clickDropper li ul li:first-child::before {
+	color: #404040;
```

Comment:
Color literals like `#404040` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs